content/en/about/feature-stages/index.md

```diff
@@ -71,7 +71,7 @@ Below is our list of existing features and their current phases. This informatio
 |-------------------|-------------------
 | [Service-to-service mutual TLS](/docs/concepts/security/#mutual-tls-authentication)         | Stable
 | [Kubernetes: Service Credential Distribution](/docs/concepts/security/#pki)   | Stable
-| [SDS based key and certificate provisioning](/docs/concepts/security/#the-istio-agent-in-kubernetes)  | Stable
+| [SDS based key and certificate provisioning](/docs/concepts/security/#pki)  | Stable
```

Comment:
+ @myidpt FYI, I change the link to this so this PR can be merged. Please take care of this later when you have chance.